tastytaps/urls.py

```diff
@@ -3,8 +3,10 @@
 
 from .views import HomePageView
 
+admin.autodiscover()
 
 urlpatterns = [
     url(r'^$', HomePageView.as_view(), name='home'),
+    url(r'api/', include('tastytaps.taps.urls')),
```

Comment:
It might be a good future-proof thing to version the api, so `r'api/v1/'` or something.